src/common/xaml.props

```diff
@@ -0,0 +1,12 @@
+<Project DefaultTargets="Build">
+  <ItemGroup>
+    <Compile Update="**\*.xaml.cs">
+      <DependentUpon>%(Filename)</DependentUpon>
+      <SubType>Code</SubType>
+    </Compile>
+    <Page Include="**\*.xaml">
```

Comment:
The reason why I included `<None Remove="**\*.xaml" />` before this in my example is that without that, when adding a new xaml control inside VS adds a new line here with `<None Remove="AddedFile.xaml" />`

But I have no idea how this behaves and what VS would do when this is in am imported project. Did you check that it works properly and doesn't modify the csproj?

Reply:
Hmm yeah you're right, it does modify the .csproj...

Reply:
Looks like the is the associated issue: https://github.com/dotnet/project-system/issues/3639

Reply:
I think for the moment I'm going to revert the change for converting our WPF projects to the new csproj and try again when 15.8 is available.

Reply:
I didn't think this was a bug. They are included as None by default so VS tries to do its best to undo that and make sure they're only included as Content.